opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -338,14 +333,13 @@ namespace Opm {
                 // chopping of the update.
                 updateState(x,reservoir_state);
                 wellModel().updateWellState(xw, well_state);
+                // if the solution is updated the solution needs to be comunicated to ebos
+                // and the cachedIntensiveQuantities needs to be updated.
+                convertInput( iteration, reservoir_state, ebosSimulator_ );
+                ebosSimulator_.model().invalidateIntensiveQuantitiesCache(/*timeIdx=*/0);
```

Comment:
that's required because the model does not assume only the last "sub-solution" (in the Taylor series sense) of the time discretization to be variable. time index 0 just means "most recent sub-solution of the current time step".